Source/DafnyCore/Options/CommonOptionBag.cs

```diff
@@ -212,7 +212,7 @@ Note that quantifier variable domains (<- <Domain>) are available in both syntax
 
   public static readonly Option<bool> TypeSystemRefresh = new("--type-system-refresh", () => false,
     @"
-false - The type-inference engine and supported types are those of Dafny 4.0.
+false (default) - The type-inference engine and supported types are those of Dafny 4.0.
```

Comment:
The generated documentation already indicates the default value, so no need to specify it explicitly. Example:
<img width="1331" alt="image" src="https://github.com/user-attachments/assets/056d7915-c960-432b-bb9b-1d763a75de14" />